### ISSUE TITLE
Publish on npm

### ISSUE THREAD

**Issue by mathroc** [T00rk/bootstrap-material-datetimepicker#108]
Could you publish the package on npmjs.com ? 
It would make it easier to install/update to the latest release in a project where dependencies are managed with npm

**Comment by elGusto** [comment]
You can install any module from github with

    npm install username/reponame#branch

**Comment by mathroc** [comment]
@elGusto yeah but that would be the last commit of the branch, not the last release

**Comment by frederikprijck** [comment]
Any updates on this ?

**Comment by frederikprijck** [comment]
Still nothing ?
